cmd/notation/verify.go

```diff
@@ -148,3 +160,20 @@ func resolveReference(ctx context.Context, opts *SecureFlagOpts, reference strin
 
 	return ref, nil
 }
+
+func printMetadataIfPresent(outcome *notation.VerificationOutcome) {
+	// ignoring error, getting metadata for a successful outcome
+	metadata, _ := outcome.GetUserMetadata()
+
+	if len(metadata) > 0 {
+		fmt.Println("\nThe artifact was signed with the following user metadata.")
+		tw := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
+		fmt.Fprintln(tw, "\nKEY\tVALUE\t")
+
+		for k, v := range metadata {
+			fmt.Fprintf(tw, "%v\t%v\t\n", k, v)
+		}
```

Comment:
should we move this logic to below file ? https://github.com/notaryproject/notation/blob/767f8ae4a762541dc6bacb531d65d38d212734c9/internal/ioutil/print.go#L11